packages/create/src/actions/createProject.ts

```diff
@@ -0,0 +1,83 @@
+import { resolve } from 'node:path';
+import { styleText } from 'node:util';
+import { existsSync, statSync, mkdirSync } from 'node:fs';
+import { isEmptyDir, cwd, getRunCommand, banner } from '@sku-lib/utils';
+import { generatePackageJson } from '../generators/packageJson.js';
+import { generateTemplateFiles } from '../generators/templates.js';
+import { installDependencies } from '../services/install.js';
+import { formatProject } from '../services/format.js';
+import { validatePackageName } from '../validation/packageName.js';
+import type { Template } from '../types/index.js';
+
+interface CreateProjectOptions {
+  projectName: string;
+  template: Template;
+}
+
+export const createProject = async ({
+  projectName,
+  template,
+}: CreateProjectOptions) => {
+  console.log(
+    `🚀 Creating new sku project: ${styleText('cyan', projectName)} with ${styleText('cyan', template)} template`,
+  );
+
+  const targetPath = resolveProjectPath(projectName);
+  validatePackageName(targetPath);
+
+  console.log(`📁 Creating project at ${styleText('cyan', targetPath)}`);
+
+  validateTargetDirectory(targetPath);
+  createProjectDirectory(targetPath);
+
+  await generatePackageJson(targetPath, { projectName, template });
+  await generateTemplateFiles(targetPath, { projectName, template });
+  await installDependencies(targetPath);
+  await formatProject(targetPath);
+
+  const nextSteps = [
+    `${styleText('cyan', 'cd')} ${projectName}`,
+    `${styleText('cyan', getRunCommand('start'))}`,
```

Comment:
Shouldn't all of `cd ${projectName}` be styled here? If so then the backticks shouldn't be necessary around the `styleText`:

```suggestion
    styleText('cyan', `cd ${projectName}`)},
    styleText('cyan', getRunCommand('start')),
```